packages/maskbook/src/_locales/en/messages.json

```
@@ -332,6 +332,13 @@
     "plugin_red_packet_description_empty": "The red packet is empty.",
     "plugin_red_packet_description_broken": "The red packet is broken.",
     "plugin_red_packet_description_failover": "From {{name}} / {{shares}} shares / {{total}} {{symbol}}",
+    "plugin_red_packet_history_duration": "Time: {{startTime}} ~ {{endTime}} (UTC+8)",
```

```suggestion
    "plugin_red_packet_history_duration": "Time: {{startTime}} ~ {{endTime}}",
```
we user not all in **UTC+8** time region

plz use user local time.